### DISCUSSION TITLE
Module Versioning

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#1597] [Q&A]
Hi,

How do I keep previous version of module source code in an external module project and deploy latest module version?

**Comment by sbwalker** [comment]
Could you please clarify your question, it not clear to me what you are trying to do. If you are trying to maintain various versions of the source code of your module I would recommend you rely on a source control system such as Git as that is what they are designed for.

**Reply by Sharafudeen** [reply]
> Could you please clarify your question, it not clear to me what you are trying to do. If you are trying to maintain various versions of the source code of your module I would recommend you rely on a source control system such as Git as that is what they are designed for.

Thanks for the answer. Could you tell me for what purpose to mention module version in the ModuleInfo file?
